code/ai/ai_profiles.cpp

```diff
@@ -580,6 +580,9 @@ void parse_ai_profiles_tbl(const char *filename)
 						Warning(LOCATION, "Invalid ai secondary range awareness mode '%s' specified", buf);
 					}
 				}
+
+				set_flag(profile, "$all ships take hull damage in collisions:", AI::Profile_Flags::All_ships_take_hull_damage_from_collisions);
```

Comment:
Would "$no shield damage from collisions:" be a better?

Reply:
That might be a bit better, yeah.